releases/vhd-notes/aks-ubuntu-1604/aks-ubuntu-1604-201904_2019.06.08.txt

```diff
@@ -0,0 +1,146 @@
+Starting build on  Sat Jun 8 16:28:33 UTC 2019
+Using kernel:
+Linux version 4.15.0-1045-azure (buildd@lcy01-amd64-006) (gcc version 5.4.0 20160609 (Ubuntu 5.4.0-6ubuntu1~16.04.10)) #49-Ubuntu SMP Mon May 13 16:30:09 UTC 2019
+Components downloaded in this VHD build (some of the below components might get deleted during cluster provisioning if they are not needed):
+  - apt-transport-https
+  - auditd
+  - blobfuse
+  - ca-certificates
+  - ceph-common
+  - cgroup-lite
+  - cifs-utils
+  - conntrack
+  - cracklib-runtime
+  - ebtables
+  - ethtool
+  - fuse
+  - git
+  - glusterfs-client
+  - init-system-helpers
+  - iproute2
+  - ipset
+  - iptables
+  - jq
+  - libpam-pwquality
+  - libpwquality-tools
+  - mount
+  - nfs-common
+  - pigz socat
+  - util-linux
+  - xz-utils
+  - zip
+  - etcd v3.2.26
```

Comment:
should we enable 3.2.26 in this PR as well?

Reply:
https://github.com/Azure/aks-engine/pull/1451